batch.php

```diff
@@ -0,0 +1,274 @@
+<?php
+
+/**
+ * @file batch.php
+ *
+ * Copyright (c) 2003-2018 Simon Fraser University
+ * Copyright (c) 2003-2018 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * @class BatchConversionTool
+ * @ingroup plugins_generic_markup
+ *
+ * @brief Handles request for articles batch conversion.
+ *
+ * @brief CLI tool to perform batch conversion tasks
+ */
+
+require_once dirname(dirname(dirname(dirname(__FILE__)))) . '/tools/bootstrap.inc.php';
+
+class BatchConversionTool extends CommandLineTool {
+	/** @var MarkupBatchConversionHelper Batch conversion helper class*/
+	protected $markupBatchConversionHelper = null;
```

Comment:
Protected members should be prefixed with a `_`.

Reply:
https://github.com/kaschioudi/ojs3-markup/pull/53/commits/0b03be41f17e45218af21f84a206b09988d1fff8